src/main/java/com/google/devtools/build/lib/rules/cpp/CcCompilationHelper.java

```diff
@@ -1398,12 +1398,16 @@ private Artifact createCompileActionTemplate(
             /* additionalBuildVariables= */ ImmutableMap.of()));
     semantics.finalizeCompileActionBuilder(configuration, featureConfiguration, builder);
     // Make sure this builder doesn't reference ruleContext outside of analysis phase.
+    SpecialArtifact dotdTreeArtifact = null;
```

Comment:
Can you add a comment here what this is for?

Reply:
Done